lib/lookup.json

```diff
@@ -405,24 +344,6 @@
     "maintainers": "kriskowal",
     "skip": "win32"
   },
-  "react": {
-    "prefix": "v",
-    "maintainers": ["acdlite", "gaearon", "sophiebits"],
-    "yarn": true,
-    "skip": [">=12", "aix", "s390", "ppc"]
```

Comment:
This seems like an important one to get working. I guess if we’ve been skipping it since Node 12 then it’s de facto already removed, but it feels like one we should add back if we can.

Reply:
Seems `>=12` was skipped even in the original commit: https://github.com/nodejs/citgm/pull/625